src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -82,7 +94,66 @@ const ApplicationForm: React.FC = () => {
 
   const saveData = () => {}
 
-  const submitData = () => {}
+  const submitData = async () => {
+    try {
+      const bodyData = new FormData()
+      bodyData.append("fname", contactFormData.fname)
+      bodyData.append("lname", contactFormData.lname)
+      bodyData.append("email", contactFormData.email || "none")
+      bodyData.append("phone", contactFormData.phone)
+      bodyData.append("age", demographicFormData.age)
+      bodyData.append("pronounCount", "1")
+      for (let i = 0; i < demographicFormData.pronouns.length; i += 1) {
+        bodyData.append("pronouns[0-N-1]", demographicFormData.pronouns[i])
+      }
+      bodyData.append("sexualityCount", "1")
+      for (let i = 0; i < demographicFormData.sexuality.length; i += 1) {
+        bodyData.append("sexuality[0-N-1]", demographicFormData.sexuality[i])
+      }
+      bodyData.append("school", demographicFormData.school)
+      bodyData.append(
+        "collegeAffiliation",
+        demographicFormData.collegeAffiliation
+      )
+      bodyData.append("eventLocation", demographicFormData.eventLocation)
+      bodyData.append("major", demographicFormData.major)
+      bodyData.append("currentStanding", demographicFormData.currentStanding)
+      bodyData.append("country", demographicFormData.country)
+      bodyData.append("whyCruzHacks", shortAnswerFormData.whyCruzHacks)
+      bodyData.append("newThisYear", priorExperienceFormData.firstCruzHacks)
+      bodyData.append(
+        "grandestInvention",
+        shortAnswerFormData.grandestInvention
+      )
+      bodyData.append("firstCruzHack", priorExperienceFormData.firstCruzHacks)
+      bodyData.append("hackathonCount", priorExperienceFormData.hackathonCount)
+      bodyData.append(
+        "priorExperience",
+        priorExperienceFormData.priorExperience
+      )
+      bodyData.append("linkedin", connectedFormData.linkedin)
+      bodyData.append("github", connectedFormData.github)
+      bodyData.append("cruzCoins", connectedFormData.cruzCoins)
+      bodyData.append("anythingElse", connectedFormData.anythingElse)
+
+      const res = await axios({
+        method: "post",
+        url: "http://127.0.0.1:5001/cruzhacks-2022-development/us-central1/application/submit",
+        data: bodyData,
+        headers: {
+          "Content-Type": "multipart/form-data",
+          Authorization: `Bearer ${token}`,
+        },
+      })
+      console.log(res)
+      if (res.status === 201) {
+        setSubmitStatus("submitted")
+      }
```

Comment:
If it's submitted can we
1. Remove saved user profile from localstorage
2. Set status of user to submitted in localstorage
3. Set Page status to page 0

Reply:
So, I was able to do number 3, but I think we should save number 1 and 2 for a different pull request because we haven't merged the onSave function yet